Sources/Controllers/Cells/OASettingsTableViewCell.m

```diff
@@ -42,10 +42,10 @@ + (CGFloat) getHeight:(NSString *)title value:(NSString *)value cellWidth:(CGFlo
 + (CGFloat) getTextViewHeightWithWidth:(CGFloat)cellWidth title:(NSString *)title value:(NSString *)value
 {
     if (!_titleTextFont)
-        _titleTextFont = [UIFont fontWithName:@"AvenirNext-Regular" size:16.0];
+        _titleTextFont = [UIFont fontWithName:@"AvenirNext-Regular" size:17.0];
```

Comment:
Change Avenir to systemFontOfSize: wherever you see it (it is legacy code that has to be replaced)